Standardised docs (#1499)
kartoza/sawps: docs/src/user/manual/login-logout/logout.md

# Logout Page Documentation

> The Logout Page is a secure and straightforward feature designed to allow users to log out of their account or session within the users application. This page provides a clean and intuitive interface for users to initiate the logout process, ensuring that their account remains protected and their session ends securely.

Reviewer: ```suggestion
The Logout Page is a secure and straightforward feature designed to allow users to log out of their account or session within the users' application. This page provides a clean and intuitive interface for users to initiate the logout process, ensuring that their account remains protected and their session ends securely.
```

## Logout page

![Logout Form](./img/logout-1.png)

> 1. **Logout Button**: To log out of the current the user session, click the `LOG OUT` button. This action will terminate the users session, ensuring they are securely logged out of the application.

Reviewer: ```suggestion
1. **Logout Button**: To log out of the current user session, users should click the `LOG OUT` button. This action will terminate the user's session, ensuring they are securely logged out of the application.
```

## Logout Process

> When the user click the `LOG OUT` button, the following actions occur:

Reviewer: ```suggestion
When the user clicks on the `LOG OUT` button, the following actions occur:
```

> 1. The system securely terminates the users current session.

Reviewer: ```suggestion
1. The system securely terminates the user's current session.
```

> 2. User will be logged out of the users account, and any temporary data related to the users session will be cleared.

Reviewer: ```suggestion
2. The user will be logged out of their account, and any temporary data related to the user's session will be cleared.
```
